### PR TITLE
Apply Feature configuration tags to suite

### PR DESCRIPTION
**Pull request by mallison**

### CODE REVIEW

File: lib/browser-runtime.ts [hunk 1]
```diff
@@ -286,7 +287,23 @@ function createStepDescription({
...
+    tags
+      .filter(looksLikeOptions)
+      .map(tagToCypressOptions)
+      .filter((tag) => {
+        return Object.keys(tag).every((key) =>
+          SUITE_CONFIGURATION_OPTIONS.includes(key)
+        );
+      })
+      .reduce(Object.assign, suiteOptions);
+  });
```
Comment (moretti) [inline lib/browser-runtime.ts]: **suggestion (if-minor):** 
```suggestion
  const suiteOptions = pickles.reduce((acc, pickle) => {
    const tags = collectTagNames(pickle.tags)
      .filter(looksLikeOptions)
      .map(tagToCypressOptions)
      .filter((tag) => Object.keys(tag).every((key) => SUITE_CONFIGURATION_OPTIONS.includes(key)));

    return Object.assign(acc, ...tags);
  }, {});
```

**Review by moretti — APPROVED** [review]
Looks good to me, should we submit a PR on https://github.com/badeball/cypress-cucumber-preprocessor?

**Comment by mallison** [comment]
> should we submit a PR on https://github.com/badeball/cypress-cucumber-preprocessor?

Yeah I think so. Their contributing guide says to "ask first" to see if it's a feature that they'll likely accept -- that's why I wrote up a feature request initially. This seems small enough that it's worth just putting into PR.

**Comment by mallison** [comment]
@moretti just realised I've a bit more work to do here as this solution will hoist any Scenario settings to the suite level!

**Pull request closed without merging**